doc/arping.xml

```diff
@@ -60,6 +60,15 @@ xml:id="man.arping">
     <emphasis remap="I">source</emphasis>.</para>
   </refsection>
 
+  <refsection xml:id="destination">
```

Comment:
IMO it is not necessary to add a separate "destination" section just to be consistent with the ping manpage.

I would rather append the following to the Description section:
```
arping supports IPv4 addresses only. For IPv6, see ndisc6(8).
```
And add ndisc6 to the "See Also" section.